crates/viewer/re_viewer/src/ui/settings_screen.rs

```diff
@@ -181,13 +181,20 @@ fn settings_screen_ui_impl(ui: &mut egui::Ui, app_options: &mut AppOptions, keep
     // Experimental features
     //
 
-    // Currently, the wasm target does not have any experimental features. If/when that changes,
-    // move the conditional compilation flag to the respective checkbox code.
-    #[cfg(not(target_arch = "wasm32"))]
-    // Currently there are no experimental features
-    if false {
+    //#[cfg(not(target_arch = "wasm32"))]
+    if true {
         separator_with_some_space(ui);
         ui.strong("Experimental features");
+
+        ui.re_checkbox(
```

Comment:
Now that we have this `CoordinateFrame` archetype, I would expect this experimental UI to always show up in the selection panel if I log a `CoordinateFrame`, probably under the archetype itself though.

Reply:
but `CoordinateFrame` itself is still experimental though 🤷